ToolBox/BovespaConverter/config.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+    //----------------------------------------
+    // Configuration for the Bovespa Converter
+    //----------------------------------------
+
+    // Input folder in which zip raw files are saved
+    "input-folder": "C:\\Users\\Alexandre\\Documents\\Data",
+
+    // Output folder in which files will be saved (usually the Lean data folder)
+    "output-folder": "C:\\Users\\Alexandre\\Documents\\GitHub\\Lean\\Data",
```

Comment:
Other config.json files in the Toolbox have:
"output-folder": "C:\\Lean\\Data",
Is that alright?
If so, I could put
"input-folder": "C:\\Data",